File: RentIT/src/main/java/com/rentit/model/Inquiry.java
package com.rentit.model;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDate;

@Data
@Builder
@AllArgsConstructor
@NoArgsConstructor
public class Inquiry {
    int inquiryId;
    int productId;
    String message;
    LocalDate timeStamp;
    int senderId;
    String senderEmail;
    String senderPhoneNumber;
    int receiverId;
    boolean viewed;
    LocalDate viewedAt;
    LocalDate rentStart;
    LocalDate rentEnd;
}

File: RentIT-Frontend/src/model/inquiry.ts
export interface Inquiry {
  inquiryId?: number;
  productId: number;
  message: string;
  timeStamp?: Date;
  senderId?: number;
  senderEmail?: string;
  senderPhoneNumber?: string;
  receiverId?: number;
  viewed?: boolean;
  viewedAt?: Date;
  rentStart: Date;
  rentEnd: Date;
}
